### PR TITLE
Yocto update fixes

### PR DESCRIPTION
**Pull request by esajaa**
Yocto update required two different changes:

    Adding support to 64 arm architecture, which required
    -- Adding aarm64 macro to gperftools stacktrace.cc
    -- Using SYSEVENTFD2 instead of SYSEVENTFD

    Building libuv with autogen instead of gyp.

Merging this allows removing 0001-aarch64-arm.patch, 0002-eventfd2.patch, 0003-build-with-autoconf.patch under each target from meta-pelion-edge. These files were introduced in follwoing PR:
PelionIoT/meta-pelion-edge#180

### CODE REVIEW

**Comment by esajaa** [comment]
@costanic @mray190 @cmonr I moved this maestro-patch PR to this branch, since it seems that the CI wasn't able to build it from the fork. It still fails, but I'm unable to discern the reason. I also removed the hard-coded host-value for now from this PR. Please check this when you a moment suitable for it, no worries if it takes some more time now.

File: processes/wakeupfd.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func (this *WakeupFd) ReadWakeup() (ret uint64, errno error) {
 // eventfd() is Linux specific
 // C def: int eventfd(unsigned int initval, int flags);
 func eventfd(initval uint, flags int) (fd int, errno error) {
-	val, _, err := syscall.RawSyscall(syscall.SYS_EVENTFD,
+	val, _, err := syscall.RawSyscall(syscall.SYS_EVENTFD2,
```
Comment (costanic) [inline processes/wakeupfd.go:80]: why is this required?  can you give some more context here?

Comment (esajaa) [reply processes/wakeupfd.go:80]: To put it short: `aarch64` doesn't have SYS_EVENTFD, so SYS_EVENTFD2 is used instead. 

If I've understood correctly, the SYS_EVENTFD already already actually calls SYS_EVENTFD2 (at least the SYS_EVENTFD doesn't have the flags variable). 

https://man7.org/linux/man-pages/man2/eventfd.2.html

**Review by costanic — APPROVED** [review]
Looks fine to me.  CI is still broken but is likely not related, and the failure is being investigated by @mray190, so once we have a passing CI we can merge.

**Review by mray190 — APPROVED** [review]
